app/router.js

```diff
@@ -58,6 +58,16 @@ Router.map(function() {
   this.route('school', { path: 'schools/:school_id'});
   this.route('assign-students', {path: '/admin/assignstudents'});
   this.route('myprofile');
+  this.route('course-rollover');
+  this.route('curriculumInventoryReports', {
```

Comment:
true dat. i'll add a check back in, and treat the inability to successfully log in on one of the ci routes as a 'known issue'